back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/invite_reminder.rb

```diff
@@ -46,7 +46,11 @@ def activity_triggers
     end
 
     def filter_recipient(users_scope, activity:, time: nil)
-      users_scope.where(id: activity.item.invitee.id)
+      if activity.item&.invitee_id
```

Comment:
Are you sure you need this change?

Reply:
Not really, I was just aligning it with the format of `InviteReceived`

Reply:
I wondered if `users_scope.none` really does anything.

Reply:
But I thinkt the chec `if activity.item&.invitee_id` is probably a good idea, no?

Reply:
Not sure, probably it doesn't matter. So, up to you.

Reply:
Just in general I try to avoid changing code I'm not sure about.

Reply:
Turns out that reverting actually causes the new tests to fail with ...
```Ruby
NoMethodError:
       undefined method `invitee_id' for nil:NilClass
```
So, it looks like I caught a bug by adding the tests, where the scope would throw an error, because it was passed `User.all` (now `User.where.not(email : nil)`) and then try to read match user id with `invitee_id` for every user.

I've replaced the new code (which mirrors the code in `InviteReceived`